autoscoper/src/ui/AutoscoperMainWindow.cpp

```diff
@@ -2101,38 +2101,52 @@ Autoscoper 1 was developed by Andy Loomis(original CUDA version) and Mark Howiso
   about_autoscoper->show();
 }
 
+QString AutoscoperMainWindow::get_sample_data_path(QString default_config_path, QString sample_type) {
+    // check if the default config path is a valid directory
+    QDir dir(default_config_path);
+    if (dir.exists() == false) {
+        // if the path does not exist, we want to use the slicer cache directory
+        // ~\AppData\Local\slicer.org\Slicer\cache\SlicerIO\Wrist'
+#ifdef WIN32
+        default_config_path = QDir::homePath() + "/AppData/Local/slicer.org/Slicer/cache/SlicerIO/"+sample_type;
```

Comment:
```suggestion
        default_config_path = QDir::homePath() + "/AppData/Local/slicer.org/Slicer/cache/SlicerIO/" + sample_type;
```

Reply:
Let's have the sub directory configured through a CMake option.

Reply:
To support loading data from either a build or install tree, considering adding a function `bool AutoscoperMainWindow::isInstalled() const`.

This function should allow to find out if the application is executed from a build or install tree.

it can be adapted from what is done in [qSlicerCoreApplicationPrivate::isInstalled](https://github.com/Slicer/Slicer/blob/01984729eca68f0417ead58d87d3358912f094d4/Base/QTCore/qSlicerCoreApplication.cxx#L639)

Reply:
@jcfr I added the [`isInstalled`](https://github.com/NicerNewerCar/Autoscoper/blob/5bb864bddf60d0347ace05a1df1ea02e6da1d085/autoscoper/src/ui/AutoscoperMainWindow.cpp#L2104-L2112) function and updated the [`get_sample_data_path`](https://github.com/NicerNewerCar/Autoscoper/blob/5bb864bddf60d0347ace05a1df1ea02e6da1d085/autoscoper/src/ui/AutoscoperMainWindow.cpp#L2114-L2135) function accordingly. 

What subdirectory should be configured from CMake? Would that be the Slicer cache dir?